src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -23,7 +29,9 @@ storiesOf('Components/Tag', module)
           tag: {
             label: text('Label', 'Link tag'),
             path: text('Url', '/example'),
+            type: select('Variable', variants, 'link'),
```

Comment:
What's the purpose of having variants types as knobs if it's already defined as stories?

Reply:
This was literally the scope of the ticket to build in this functionality.

Reply:
Hi @papegaill , thanks for the review, your concern is clearly reasonable, this new select will let you switch between the different variants you have for the component which are indeed matching the stories we demo.
But if i have to consider what is redundant here, i'd say it's the "multiplication" of the stories for something that is actually just a parameter switch, not the availability of the parameter really handling the switch.
At the same time, i see that sometimes stories are a more evident way to show the different "variants" available per component, but we have to make it clear what is the real difference between them.
The label should be "variant", though, for this to happen, "variable" is not really identifying what the parameter is about

Reply:
@planctus I think it's confusing, because for example if on the "_as a link_" story i switch variant, I'm then able to set "_url_" on a button or a removable tag. same with parameter "_aria-label_" available only on removable tag.

Reply:
ok, you convinced me, let's remove it ;)